{{cookiecutter.project_name}}/pyproject.toml

```diff
@@ -105,58 +101,9 @@ ignore = [
     "D106", # Missing docstring in public nested class
     "D107", # Missing docstring in __init__
     "D401", # First line should be in imperative mood
-    "N818", # exception name should be named with an Error suffix
     "P103", # other string does contain unindexed parameters
     "W291", # trailing whitespace
     "W503", # line break before binary operator
-    "DAR101", # Missing parameter(s) in Docstring
-    "DAR101", # Missing parameter(s) in Docstring
-    "DAR201", # Missing "Returns" in Docstring
-    "DAR401", # Missing exception(s) in Raises section
-    "WPS110", # Found wrong variable name
-    "WPS111", # Found too short name
-    "WPS112", # Found private name pattern
-    "WPS115", # Found upper-case constant in a class
-    "WPS120", # Found regular name with trailing underscore
-    "WPS121", # Found usage of a variable marked as unused
-    "WPS122", # Found all unused variables definition
-    "WPS226", # Found string literal over-use
-    "WPS305", # Found `f` string
-    "WPS306", # Found class without a base class
-    "WPS237", # Found a too complex `f` string
-    "WPS323", # Found `%` string formatting
-    "WPS326", # Found implicit string concatenation
-    "WPS329", # Found useless `except` case
-    "WPS331", # Found variables that are only used for `return`:
-    "WPS337", # Found multiline conditions
-    "WPS338", # Found incorrect order of methods in a class
-    "WPS347", # Found vague import that may cause confusion
-    "WPS348", # Found a line that starts with a dot
-    "WPS360", # Found an unnecessary use of a raw string
-    "WPS404", # Found complex default value
-    "WPS407", # Found mutable module constant
-    "WPS412", # Found `__init__.py` module with logic
-    "WPS420", # Found wrong keyword
-    "WPS421", # Found wrong function call
-    "WPS428", # Found statement that has no effect (for `...`)
-    "WPS432", # Found magic number
-    "WPS433", # Found nested import
-    "WPS437", # Found protected attribute usage
-    "WPS440", # Found block variables overlap
-    "WPS441", # Found control variable used after block
-    "WPS453", # Found executable mismatch: shebang is present but the file is not executable
-    "WPS458", # Found imports collision
-    "WPS529", # Found implicit `.get()` dict usage
-    "WPS531", # Found simplifiable returning `if` condition in a function
-    "WPS602", # Found using `@staticmethod`
-    "WPS604", # Found incorrect node inside `class` body
-    "WPS608", # Found incorrect `super()` call: remove arguments
-    "WPS615", # Found unpythonic getter or sette
-]
-per-file-ignores = [
-    "**/settings/*.py:S101",
```

Comment:
S101 is rule for `flake8-strict`, which is not a pinned dev dependency
It is not maintained, latest release in 2017 - https://github.com/smarkets/flake8-strict/releases